premarum-web-client/utility/helpers/selectionToRow.ts

```diff
@@ -17,9 +17,24 @@ export async function GetRows(selections: IPreEnrollmentSelectionResponse[]) {
             professors.push(selections[i].professors[k].name)
         }
 
-        result.push({id: parseInt(i), entryId: selections[i].id, course: selections[i].course.courseCode, section: selections[i].sectionName,
-            credits: selections[i].course.courseCredit, days: days.join(", "), classroom: selections[i].classRoom,
-            timeslot: times.join(", "), professor: professors.join(", ")})
+        // result.push({id: parseInt(i), entryId: selections[i].id, course: selections[i].course.courseCode, section: selections[i].sectionName,
+        //     credits: selections[i].course.courseCredit, days: days.join(", "), classroom: selections[i].classRoom,
+        //     timeslot: times.join(", "), professor: professors.join(", ")})
+
+        times = [...new Set(times)]; // remove duplicates
+
+        result.push({
+            id: parseInt(i), 
+            entryId: selections[i].id, 
+            course: selections[i].course.courseCode, 
+            section: selections[i].sectionName,
+            credits: selections[i].course.courseCredit, 
+            classroom: selections[i].classRoom,
+            timeslot: days.join(", ") + "\n" + times.join(", "), 
```

Comment:
I was thinking this format would be good: "Monday 10:30-11:30, Tuesday 11:50-12:50" Or we can also abbreviate to Mon Tue, ...